packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1260,6 +1260,23 @@ export const RewriteWithBodyUsed = {
 		'Astro.rewrite() cannot be used if the request body has already been read. If you need to read the body, first clone the request.',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * `Astro.rewrite()` can't be used to rewrite an on-demand route with a static route, when using the `"server"` output.
+ *
+ */
+export const ForbiddenRewrite = {
+	name: 'ForbiddenRewrite',
+	title: "Can't use `Astro.rewrite()` from a on-demand route to static route with 'server' output.",
```

Comment:
```suggestion
	title: "Can't use `Astro.rewrite()` from an on-demand route to static route with 'server' output.",
```
typo fix here at the very least, but in fact, see suggestion for title below also.